ccd-definition/FixedLists.json

```diff
@@ -44,73 +44,73 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "GroundsList",
-    "ListElementCode": "noCare",
+    "ListElementCode": "NO_CARE",
     "ListElement": "Not receiving care that would be reasonably expected from a parent"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "GroundsList",
-    "ListElementCode": "beyondControl",
+    "ListElementCode": "BEYOND_CONTROL",
     "ListElement": "Beyond parental control"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "HearingTypeFixedList",
-    "ListElementCode": "Contested interim care order",
+    "ListElementCode": "CONTESTED_INTERIM_CARE_ORDER",
     "ListElement": "Contested interim care order"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "HearingTypeFixedList",
-    "ListElementCode": "Urgent preliminary case management heari",
+    "ListElementCode": "URGENT_PRELIMINARY_CASE_MANAGEMENT_HEARI",
```

Comment:
```suggestion
    "ListElementCode": "URGENT_PRELIMINARY_CASE_MNGMNT_HEARING",
```

Reply:
What about MGMT instead of MNGMNT?

Reply:
I think either is fine, MNGMNT to me looks more like MANAGEMENT though. And still fits in 40 charcters. @geek-soft thoughts?

Reply:
I think `MGMT` is official abbreviation for MANAGEMENT ;)